Tasks/google-play-release/GooglePlay.ts

```diff
@@ -768,6 +776,25 @@ function resolveGlobPaths(path: string): string[] {
     return [];
 }
 
+/**
+ * Get obb file for the apk from the parent directory.
+ * @returns {string[]} path of the obb file if present else null
+ */
+function getObbFromParentDirectroy(apkPath: string): string {
```

Comment:
Nit: just a small typo here - getObbFromParentDirectory

Reply:
Corrected this.